apps/server/src/modules/common-cartridge/service/common-cartridge-import.service.spec.ts

```diff
@@ -8,13 +8,24 @@ import { CommonCartridgeImportService } from './common-cartridge-import.service'
 
 jest.mock('../import/common-cartridge-file-parser', () => {
 	const fileParserMock = createMock<CommonCartridgeFileParser>();
+	const rootId = faker.string.uuid();
 
 	fileParserMock.getTitle.mockReturnValue(faker.lorem.words());
 	fileParserMock.getOrganizations.mockReturnValue([
 		{
 			pathDepth: 0,
 			title: faker.lorem.words(),
 			path: faker.system.filePath(),
+			identifier: rootId,
+			isInlined: true,
+			isResource: false,
+			resourcePath: faker.system.filePath(),
+			resourceType: faker.lorem.word(),
+		},
+		{
+			pathDepth: 1,
+			title: faker.lorem.words(),
+			path: `${rootId}/faker.system.filePath()`,

```

Comment:
I think that should in an expression
```suggestion
			path: `${rootId}/${faker.system.filePath()}`,
```